spec/unit/scrum/sprint_cleaner_spec.rb

```diff
@@ -7,9 +7,19 @@
     expect(subject).to be
   end
 
-  it 'moves remaining cards to target board', vcr: 'sprint_cleanup', vcr_record: false do
-    expect(STDOUT).to receive(:puts).exactly(13).times
-    expect(subject.cleanup('7Zar7bNm', '72tOJsGS')).to be
+  context 'given labelled flag is true' do
+    before do
+      allow(Trello::Label).to receive(:create)
+      allow_any_instance_of(Trello::Card).to receive(:add_label)
+    end
+
+    it 'moves remaining cards to target board', vcr: 'sprint_cleanup', vcr_record: false do
+      expect(STDOUT).to receive(:puts).exactly(13).times
+      expect(subject.cleanup('7Zar7bNm', '72tOJsGS', true)).to be
+      subject.sprint_board('7Zar7bNm').backlog_list.cards.each do |card|
```

Comment:
does this work? now we're expecting all the instances of card to receive the add label call which actually adds the labels.

Reply:
@Ana06 ?